documentation/API.md

```diff
@@ -124,6 +81,92 @@ Basic login endpoint that will also return (on success):
 }
 ```
 
+#### Get preferences endpoint:
+
+GET: `/get-prefs`
+
+Provides current preferences and language options for the current installation. Called by front-end at the very start of loading in order to configure the environment according to system settings.
+
+#### Get language endpoint:
+
+GET: `/language/<langauge-code>`
```

Comment:
```suggestion
GET: `/language/<language-code>`
```

Reply:
Example would be a good thing to also add here

Reply:
Also - tried getting for `en_dtac` and it wasn't there, but the return is 404. Should it be some other error maybe?

Reply:
Great. Seems ok, I thought some specific message as: this language isn't included, but I guess 404 is alright.

You still have the typo to correct on this one, please.